From 3813f00ac64ba684ba99a2f64b67ca114ea96c52 Mon Sep 17 00:00:00 2001
From: Hains van den Bosch <hainsvdbosch@ziggo.nl>
Date: Sat, 18 Aug 2018 22:05:20 +0200
Subject: [PATCH] gstaudiobuffersplit.c: change datatype to long long unsigned

To fix build.

../../../git/gst/audiobuffersplit/gstaudiobuffersplit.c:576:27: error: format '%lu' expects argument of type 'long unsigned int',
but argument 8 has type 'guint64' {aka 'long long unsigned int'} [-Werror=format=]
---
 gst/audiobuffersplit/gstaudiobuffersplit.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gst/audiobuffersplit/gstaudiobuffersplit.c b/gst/audiobuffersplit/gstaudiobuffersplit.c
index 418400c12..858e6c43d 100644
--- a/gst/audiobuffersplit/gstaudiobuffersplit.c
+++ b/gst/audiobuffersplit/gstaudiobuffersplit.c
@@ -573,7 +573,7 @@ gst_audio_buffer_split_clip_buffer_start_for_gapless (GstAudioBufferSplit *
 
   nsamples = gst_buffer_get_size (buffer) / bpf;
 
-  GST_DEBUG_OBJECT (self, "Have to drop %lu samples, got %u samples",
+  GST_DEBUG_OBJECT (self, "Have to drop %llu samples, got %u samples",
       self->drop_samples, nsamples);
 
   if (nsamples <= self->drop_samples) {
-- 
2.17.1

